src/Events/Controllers/AppEventsController.cs

```diff
@@ -217,6 +215,29 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEventRequestModel c
             }
         }
 
+        private static void ValidateQueryParams(string after, DateTime? from, DateTime? to, int size)
+        {
+            if (string.IsNullOrEmpty(after) && from == null)
+            {
+                throw new ArgumentException("From or after must be defined.");
+            }
+
+            if (from != null && from.Value.Kind == DateTimeKind.Unspecified)
```

Comment:
Do we support non-UTC datetimekinds?

Reply:
Yes, but if we want to prevent a variation of timezones being used stored in the database we might need to actively change the zone to UTC. I haven't thought of it before.

Reply:
What is specified in Swagger for now is that the time should be defined in UTC, but I guess we can change this is we want. Converting it to UTC before passing it on to the service is an easy task. 
Before we have defined that Platform only uses UTC, and that conversion to local time should be done by the clients / app when required